cypress/integration/unit/cancel-editing-document.spec.js

```diff
@@ -24,7 +24,7 @@ function uploadFileToCancel(file) {
 
 function openAgendaitemDocumentBatchEdit() {
   cy.intercept('GET', '/document-types?**').as('getDocumentTypes');
-  cy.intercept('GET', '/access-levels?**').as('getAccessLevels');
```

Comment:
this might not do the trick.
the intercept is very vague. Might no longer be needed though, need to check.